R/datasets.R

```diff
@@ -15,57 +15,45 @@
 #' @include schema.R
 NULL
 
-#' Reads a tabular Foundry dataset as data.frame.
+#' Reads a tabular Foundry dataset as data.frame or an arrow Table.
```

Comment:
The hierarchy in this library makes a lot of sense to me (i.e. wrapper around low-level client, with one client per API namespace)